tests/test_merge_single_band.py

```diff
@@ -0,0 +1,29 @@
+import sys
+from pathlib import Path
+
+import numpy as np
+import xarray as xr
+
+# Try import normally first (works when package installed or editable)
+try:
+    from metnet.data.merge_single_band import merge_086um
+except ModuleNotFoundError:
+    ROOT = str(Path(__file__).resolve().parents[1])
+    if ROOT not in sys.path:
+        sys.path.insert(0, ROOT)
+    from metnet.data.merge_single_band import merge_086um
```

Comment:
This should just be the import, if it can't be found, then the metnet package is installed incorrectly, and it should fail to try to run.
```suggestion
from metnet.data.merge_single_band import merge_086um
```